src/Og.php

```diff
@@ -117,7 +116,10 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
    *  the OG membership ID and the group ID as the value. If nothing found,
    *  then an empty array.
    */
-  public static function getEntityGroups($entity_type, $entity_id, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+  public static function getEntityGroups(EntityInterface $entity, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
+    $entity_type = $entity->getEntityTypeId();
```

Comment:
OK. I think I've been using the _id suffixed versions. So go with that??
On 1 Dec 2015 10:50, "Amitai Burstein" <notifications@github.com> wrote:

> In src/Og.php
> <https://github.com/amitaibu/og/pull/53#discussion_r46264240>:
>
> > @@ -117,7 +116,10 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
> >     *  the OG membership ID and the group ID as the value. If nothing found,
> >     *  then an empty array.
> >     */
> > -  public static function getEntityGroups($entity_type, $entity_id, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
> > +  public static function getEntityGroups(EntityInterface $entity, $states = [OG_STATE_ACTIVE], $field_name = NULL) {
> > +    $entity_type = $entity->getEntityTypeId();
>
> I don't mind too much
>
> Me neither, just want to be consistent. I could go with $entity_type_id
> as-well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/53/files#r46264240>.
>